lib/utils.js

```diff
@@ -496,3 +496,20 @@ class Where extends SequelizeMethod {
 exports.Where = Where;
 
 exports.validateParameter = parameterValidator;
+
+
+exports.mapIsolationLevelStringToTedious = isolationLevel => {
+  const tediousIsolationLevel = require('tedious').ISOLATION_LEVEL;
```

Comment:
@sushantdhiman I've used the latter approach. Since its a private method, we are throwing error incase tedious lib instance is not passed to Utils function.